ipaserver/install/cainstance.py

```diff
@@ -1953,7 +1953,14 @@ def import_included_profiles():
         cn=['certprofiles'],
     )
 
-    api.Backend.ra_certprofile.override_port = 8443
+    try:
+        with api.Backend.ra_certprofile as profile_api:
+            print(profile_api.ca_host)
+            if profile_api.ca_host == api.env.host:
```

Comment:
It would be nice to add a comment explaining why the port is overriden.

Reply:
print() removed and giant comment block added.